components/CardContent.js

```diff
@@ -10,10 +10,14 @@ const propTypes = {
   cardLink: PropTypes.node,
   cardInfo: PropTypes.array,
   className: PropTypes.string,
+  iconDescription: PropTypes.string,
 };
 
 const defaultProps = {
+  iconDescription: 'card icon',
   cardIcon: 'app-services',
+  cardTitle: 'card title',
+  cardInfo: ['info']
```

Comment:
Yeah I think you're good to leave it out - in any scenario, for it to render the user has to pass it in as a prop